src/renderer/services/neo.js

```diff
@@ -54,8 +61,12 @@ export default {
       }
 
       try {
+        const dataBTCgen = wallets.generateBTCwalletKey();
         const account = new wallet.Account(wallet.generatePrivateKey());
         const encryptedWIF = wallet.encrypt(account.WIF, passphrase);
+        const mnemonic = dataBTCgen.key.mnemonic;
+        const encryptedMnemonic = CryptoJS.AES.encrypt(mnemonic, account.WIF);
```

Comment:
There is the reason for that, Suppose a user uses the "Login with WIF" method, then it should also show the same BTC address, So If we use there passphrase, then the function will not get the passphrase and will create new BTC address for the user.

Reply:
hmm, if the user uses "Login with WIF" how would it be able to get the same BTC address repeatedly anyway? Consider if a different computer is used from "Login with WIF they would have to also enter the mneumonic, it would not know it from the WIF. I guess it is nice at least for it to get the same btc address on the same computer though.